tests/test_fit_result.py

```diff
@@ -37,81 +85,96 @@ def ge_constraint(a):  # Has to be in the global namespace for pickle.
         # Added None to check the default objective
         subclasses(BaseMinimizer) - {TrustConstr, COBYLA, ChainedMinimizer, DifferentialEvolution, MINPACK} | {None, (BFGS, NelderMead)}
     )
-@pytest.mark.parametrize('fit_kwargs, expected_par, expected_std, expected_obj, expected_got',
+@pytest.mark.parametrize('fit_kwargs, expected_par, expected_std, expected_obj, expected_gof',
     [
         # No specific objective
-        (dict(x=np.linspace(1, 10, 10), y=3 * np.linspace(1, 10, 10) ** 2), {a: 3, b: 2}, 
-         {a: 0, b: 0}, LeastSquares, {'r_squared': 1.0, 'objective_value': 1e-23, 'chi_squared': 1e-23}),
+        (dict(x=np.linspace(1, 10, 10), y=3 * np.linspace(1, 10, 10) ** 2),
+         {a: 3, b: 2}, {a: 0, b: 0},
+         LeastSquares,
+         {'r_squared': 1.0, 'objective_value': 1e-23, 'chi_squared': 1e-23}),
         # Test objective LeastSqueares
-        (dict(objective=LeastSquares, x=np.linspace(1, 10, 10), y=3 * np.linspace(1, 10, 10) ** 2), {a: 3, b: 2}, 
-         {a: 1e-13, b: 1e-13}, LeastSquares, {'r_squared': 1.0, 'objective_value': 1e-23, 'chi_squared': 1e-23}),
+        (dict(objective=LeastSquares, x=np.linspace(1, 10, 10), y=3 * np.linspace(1, 10, 10) ** 2),
+         {a: 3, b: 2}, {a: 1e-13, b: 1e-13},
+         LeastSquares,
+         {'r_squared': 1.0, 'objective_value': 1e-23, 'chi_squared': 1e-23}),
         # Test objective LogLikelihood
-        (dict(objective=LogLikelihood, x=np.linspace(1, 10, 10)), {a: 62.56756, b: 758.08369}, # TODO Adjust x_data
-         {a: float('nan'), b: float('nan')}, LogLikelihood, 
+        (dict(objective=LogLikelihood, x=np.linspace(1, 10, 10)),  # TODO Adjust x_data
```

Comment:
Still TODO?

Reply:
Kind of. This PR shouldn't be merged before we have good data for LL so that every test passes.